vernac/vernacentries.ml

```diff
@@ -2681,12 +2685,6 @@ let rec interp_expr ?proof ~atts ~st c =
   (* This one is possible to handle here *)
   | VernacAbort id    -> CErrors.user_err  (str "Abort cannot be used through the Load command")
```

Comment:
Why is that btw?

Reply:
What do you exactly mean? The comment refers that in this case, we could just drop the proof state, however the STM may still think the proof is open.

Reply:
`VernacAbort None` is just `VtCloseProof (fun lemma:_ -> ())` no?

Reply:
Yup, but there is some special handling in the upper layers which I'm not very familiar with.